teps/0104-tasklevel-resource-requirements.md

```diff
@@ -118,63 +118,69 @@ spec:
   resources:
```

Comment:
a check for the augment on the existing [`Task.spec.resources`](https://github.com/tektoncd/pipeline/blob/3c0fb3b8d0badf58a429f890420e0d44faa0e36c/pkg/apis/pipeline/v1beta1/task_types.go#L87) field which contains `inputs` and `outputs` for `TaskResources`  

thinking perhaps the context will be a bit diff - as `inputs/outputs` for running the actual build task while the added `limits/requests` for running the beneath Pod. but after checking other existing `resources` fields, like `Task.Steps.*` etc, think still should be a better choice to keep the same field name and add the `limits` and `requests`, as for a consistent naming on the same funcs.

Reply:
Yeah, unfortunately `resources` is an overloaded word here. However, I agree that it makes sense to use it in this context, because it is the terminology Kubernetes uses to describe compute resources, and luckily PipelineResources will be going away which should lessen the confusion.

Reply:
Hmmm...that's unfortunate...would the limits and requests fields be added under the current `resources` field?

Reply:
ah I see the problem, this is definitely annoying but I'm hoping naming bikeshedding won't block this PR-- maybe we could call it "compute" or something

Reply:
an updates from implementation... tentatively I keep the added `requests` and `limits` under the mentioned `resources` field for this moment 
https://github.com/tektoncd/pipeline/pull/4877/files?diff=unified&w=0#diff-f67e6d3007cea84a7a2e6301beb54c39b96b75ac9e57d888cbf5cc86c57510f3R77

thinking these reasons:
- a consistent naming for (container computing) resources requirements as being used in all other `resources` field, like `Task.spec.step.resources`, `Task.spec.stepTemplate.resources` 
- comments on code level and docs updates can be done to clear the minor(but meaningful) context diffs between resources for building steps and container computing resources